applications/mail/src/app/helpers/transforms/transformLinks.ts

```diff
@@ -51,7 +51,7 @@ const sanitizeRelativeHttpLinks = (link: HTMLLinkElement) => {
         // link.href is the absolute value of the link: mail.proton.me is prepended, use getAttribute
         const url = link.getAttribute('href');
 
-        if (url) {
+        if (url && !url.toLowerCase().startsWith('hxxps')) {
```

Comment:
Does the customization in this function make any sense at all? A URL is always passed **without** a protocol and if this starts with hxxps, then in my opinion the http:// should still be added.

Example `hxxps.example.com` should become `http://hxxps.example.com`. With your adjustment it will not longer work.